Client/Frontend/Browser/Tabs/RemoteTabs/RemoteTabsPanel.swift

```diff
@@ -81,20 +110,28 @@ class RemoteTabsPanel: UIViewController,
         tableViewController.refreshTabs(state: state)
     }
 
-    func forceRefreshTabs() {
+    // MARK: - Redux
+
+    private func forceRefreshTabs() {
         tableViewController.refreshTabs(state: state, updateCache: true)
     }
 
-    @objc
-    func notificationReceived(_ notification: Notification) {
-        let name = notification.name
-        if name == .FirefoxAccountChanged || name == .ProfileDidFinishSyncing {
-            ensureMainThread {
-                self.tableViewController.refreshTabs(state: self.state)
-            }
-        }
+    private func subscribeRedux() {
+        guard isReduxIntegrationEnabled else { return }
+        store.dispatch(RemoteTabsPanelAction.panelDidAppear)
+        store.dispatch(ActiveScreensStateAction.showScreen(.remoteTabsPanel))
```

Comment:
@OrlaM @yoanarios A few areas in particular I was hoping to get your 👀 on. Here I'm not sure whether we should be calling the `.showScreen` action.

I added it because it seems like it should be called whenever the screen becomes active (and continues to need updates) but in `ThemeSettingsController` it appears to only be called when changing a setting, not when the UI is first displayed (afaics).

Reply:
The showScreen needs to be trigger first so the `ActiveScreensState` reducer can forward the actions to the `ScreenState` reducer